reddwarf/utils/consensus.py

```diff
@@ -35,7 +35,7 @@ class ConsensusResult(TypedDict):
 # order to avoid filtering by that measure.
 def select_consensus_statements(
     vote_matrix: VoteMatrix,
-    mod_out_statement_ids: list[int] = [],
+    mod_out_statement_ids: list[int | str] = [],
```

Comment:
```suggestion
    mod_out_statement_ids: list[int] | list[str] = [],
```

I've recently learned that this suggestion is more the semantic we'd want -- either all strings or all ints, not a list of either/or. (there may be other places in the codebase where I did it the other way, but i'd prefer this way going fwd :) )

Reply:
thank you very much!